src/flatpickr.js

```diff
@@ -1801,8 +1801,9 @@ function Flatpickr(element, config) {
 				.map(dObj => self.formatDate(dObj, self.config.altFormat))
 				.join(joinChar);
 		}
-
-		triggerEvent("ValueUpdate");
+		if (triggerChange !== false) {
```

Comment:
This part is unclear to me.
`triggerChange` shouldn't prevent `valueUpdate` from getting triggered